zipline-cli/src/main/kotlin/app/cash/zipline/cli/GenerateKeyPair.kt

```diff
@@ -0,0 +1,46 @@
+/*
+ * Copyright (C) 2022 Block, Inc.
+ *
+ * Licensed under the Apache License, Version 2.0 (the "License");
+ * you may not use this file except in compliance with the License.
+ * You may obtain a copy of the License at
+ *
+ *      http://www.apache.org/licenses/LICENSE-2.0
+ *
+ * Unless required by applicable law or agreed to in writing, software
+ * distributed under the License is distributed on an "AS IS" BASIS,
+ * WITHOUT WARRANTIES OR CONDITIONS OF ANY KIND, either express or implied.
+ * See the License for the specific language governing permissions and
+ * limitations under the License.
+ */
+
+package app.cash.zipline.cli
+
+import app.cash.zipline.cli.GenerateKeyPair.Companion.NAME
+import app.cash.zipline.loader.internal.generateKeyPair
+import java.io.PrintStream
+import picocli.CommandLine.Command
+
+@Command(
+  name = NAME,
+  description = [
+    "Generate an Ed25519 key pair for ManifestSigner and ManifestVerifier"
+  ],
+  mixinStandardHelpOptions = true,
+  versionProvider = Main.VersionProvider::class,
+)
+class GenerateKeyPair : Runnable {
+  override fun run() {
+    run(System.out)
```

Comment:
Clikt lets you do constructor DI. Just sayyyiiinnn.

Reply:
Oooh nice. Clikt is so good.

Reply:
I defined a constructor parameter with a default value and it worked nicely.